forecast-admin/forecast/opportunities/management/commands/load_opportunities.py

```diff
@@ -0,0 +1,220 @@
+
+        adv = cls.parse_advisor(row[23])
+        advisor = cls.insert_advisor(adv[0], adv[1], adv[2])
+
+        opportunity = cls.model(
+            office=office,
+            description=row[3],
+            place_of_performance_city=row[4],
+            place_of_performance_state=row[5],
+            naics=row[6],
+            socioeconomic=row[7],
+            contract_type=row[8],
+            procurement_method=row[9],
+            competition_strategy=row[10],
+            dollar_value_min=cls.parse_dollars(row[11]),
+            dollar_value_max=cls.parse_dollars(row[12]),
+            delivery_order_value=row[13],
+            incumbent_name=row[14],
+            new_requirement=row[16],
+            funding_agency=row[17],
+            estimated_solicitation_date=cls.parse_date(row[18]),
+            fedbizopps_link=row[19],
+            estimated_fiscal_year=row[20].split("-")[0][3:],
+            estimated_fiscal_year_quarter=int(row[20].split("-")[1][0]
+                                                     .replace(r"Q", "0")),
+            additional_information=row[24],
+            published=True
+        )
+        return opportunity
+    #     schedule = row[9]
+    #     if schedule != cls.schedule_name:
+    #         raise ValueError('skipping schedule: {}'.format(schedule))
+    #
+    #     price = row[5]
+    #     if not price:
+    #         raise ValueError('missing price')
+    #
+    #     price = cls.model.normalize_rate(price)
+    #     display_price = price if price >= FEDERAL_MIN_CONTRACT_RATE else None
+    #
+    #     contract = cls.model(
+    #         idv_piid=row[6],
+    #         contract_start=cls.parse_date(row[12]),
+    #         contract_end=cls.parse_date(row[13]),
+    #         contract_year=cls.int_or_fallback(row[11], 1),
+    #         vendor_name=row[7],
+    #         price_min=row[X].replace("\$","").replace(",","")
+    #         price_max=row[X].replace("\$","").replace(",","")
+    #         labor_category=row[1].strip().replace('\n', ' '),
+    #         education_level=cls.model.get_education_code(row[2]),
+    #         min_years_experience=cls.int_or_fallback(row[3]),
+    #         hourly_rate_year1=price,
+    #         hourly_rate_year2=None,
+    #         hourly_rate_year3=None,
+    #         hourly_rate_year4=None,
+    #         hourly_rate_year5=None,
+    #         current_price=display_price,
+    #         next_year_price=None,
+    #         second_year_price=None,
+    #         contractor_site=row[10],
+    #         schedule=schedule,
+    #         business_size=row[8],
+    #         sin=row[0]
+    #     )
+    #
+    #     contract.full_clean(exclude=['piid'])
+    #
+    #     return contract
+    #
+    # @staticmethod
+    # def int_or_fallback(x, fallback=0):
+    #     try:
+    #         return int(x)
+    #     except ValueError:
+    #         return fallback
+    #
+
+    @staticmethod
+    def parse_date(s):
```

Comment:
I think that, in a prior review, we thought this could be replaced with strftime.